network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/airtable_block.html

```diff
@@ -1 +1,10 @@
-<script async defer src="https://static.airtable.com/js/embed/embed_snippet_v1.js"></script><iframe class="airtable-embed mb-4 airtable-dynamic-height" src="{{ value.url }}" frameborder="0" onmousewheel="" width="100%" height="{{ value.height }}" style="background: transparent; border: 1px solid #ccc;"></iframe>
+<div class="container {{block.block_type}}-block">
+    <div class="row justify-content-center no-gutters">
+        <div class="streamfield-content">
+            <script async defer src="https://static.airtable.com/js/embed/embed_snippet_v1.js"></script><iframe
+                class="airtable-embed mb-4 airtable-dynamic-height" src="{{ value.url }}" frameborder="0"
+                onmousewheel="" width="100%" height="{{ value.height }}"
```

Comment:
why is there a 1998 style `onmousewheel` here that does nothing?

Reply:
Not sure! It was there from the master branch, but since it does nothing I removed it in latest commit.